src/style.css

```diff
@@ -0,0 +1,267 @@
+}
+
+.header {
+  /* display: flex; */
+  align-items: center;
+  justify-content: space-between;
+  position: unset;
+
+  /* background-color: #fff; */
+
+  /* box-shadow: 1px 1px 4px 0 rgb(0 0 0 / 10%); */
+  width: 100%;
+  z-index: 3;
+  height: 0;
+  margin-bottom: 40px;
+
+  /* margin-bottom: 100px; */
+}
+
+.max-width-header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  max-width: 1350px;
+  margin: auto;
+}
+
+.header .logo {
+  display: block;
+  float: left;
+  font-size: 27px;
+  padding: 10px 20px;
+  margin-top: 6.5px;
+  text-decoration: none;
+  color: var(--secondary-color);
+  font-style: normal;
+}
+
+a {
+  color: #344563;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+  font-weight: 400;
+}
+
+.desktopMenu-up {
+  display: block;
+}
+
+.topnav {
+  width: fit-content;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+}
+
+.topnav a {
+  text-decoration: none;
+  font-size: 16px;
+  padding: 7px 16px;
+}
+
+.select-tag {
+  appearance: none;
+  outline: 0;
+  border: 0;
+  box-shadow: none;
+  flex: 1;
+  padding: 0 1em;
+  color: #fff;
+  background-color: #0365ac;
+  background-image: none;
+  cursor: pointer;
+  font-size: 16px;
+}
+
+.select {
+  position: relative;
+  display: flex;
+  width: 20em;
+  height: 3em;
+  border-radius: 0.25em;
+  overflow: hidden;
+  margin-right: 20px;
+}
+
+.select::after {
+  content: "\25BC";
+  position: absolute;
+  top: 0;
+  right: 0;
+  padding: 1em;
+  background-color: #024e85;
+  transition: 0.25s all ease;
+  pointer-events: none;
+}
+
+/* Transition */
+.select:hover::after {
+  color: #29a3fb;
+}
+
+.banner {
+  width: 100%;
+  height: 65vh;
+  background-image: url("https://www.themealdb.com/images/category/chicken.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  display: flex;
+  align-items: flex-end;
+  justify-content: center;
+  margin-bottom: 230px;
+}
+
+.banner h1 {
+  font-size: 45px;
+  color: #dbdbdb;
+  font-family: "Countryside", sans-serif;
+  height: 30%;
+  margin: 0;
+}
+
+.banner span {
+  height: 19%;
+  margin-top: 60px;
+  color: var(--secondary-color);
+}
+
+.innerAligment {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  max-width: 550px;
+  min-width: 250px;
+  margin: 30px;
+  text-align: center;
+  margin-bottom: -70px;
+  height: 140px;
+}
+
+.cards {
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: row;
+  grid-gap: 1rem;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.flex-card {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  max-width: 100%;
+  transition-property: border-color, box-shadow;
+  transition-duration: 1.5s;
+}
+
+.align_images {
+  margin: auto;
+}
+
+.imagen_proyectos {
+  width: 100%;
+  border-radius: 8px;
+}
+
+.text_child h2 {
+  font-weight: 700;
+  line-height: 25px;
+  line-height: 35px;
+  margin: 0;
+  margin-top: 15px;
```

Comment:
This how margin works, sure you are familiar with it :), `margin: top right bottom left;` 
So we could improve this by making it like this `margin: 15px 0 0;` the left will inherit from the right the `0` value
What do you think?